test/isInt.js

```diff
@@ -0,0 +1,40 @@
+var PushStateTree = require('../src/pushStateTree');
+
+describe('PushStateTree isInt', function() {
+
+  it('should return true for 0', function() {
+    expect(PushStateTree.isInt(0)).to.be.true;
+  });
+
+  it('should return true for 1', function() {
+    expect(PushStateTree.isInt(1)).to.be.true;
+  });
+
+  it('should return true for "0"', function() {
```

Comment:
`should return true for valid number values in strings`, then you can expect true for '0' and '1' in the same test.